.github/workflows/deploy.yml

```diff
@@ -0,0 +1,48 @@
+name: Build Django-RQL library
```

Comment:
Name should be: Deploy Django-RQL library

Reply:
Awesome! Updated